### PR TITLE
Added comment to document the usage of the custom_css parameter in th…

### PR DESCRIPTION
**Pull request by onepointconsulting**
Added comment to document the usage of the custom_css parameter in the generated config.toml. This can be helpful for developers to find the parameter.

### CODE REVIEW

File: backend/chainlit/config.py [hunk 1]
```diff
@@ -64,6 +64,9 @@
 # Link to your github repo. This will add a github button in the UI's header.
 # github = ""
 
+# Specify a CSS file that can be used to customize the user interface.
+# custom_css = '/assets/test.css'
```
Comment (willydouhard) [inline backend/chainlit/config.py:68]:
```suggestion
# custom_css = '/public/test.css'
```

I think most people will serve the css either from the public dir or an external link

Comment (onepointconsulting) [reply backend/chainlit/config.py]: I have updated the pull request. Please check.

File: backend/chainlit/config.py [hunk 2]
```diff
@@ -65,7 +65,8 @@
 # github = ""
 
 # Specify a CSS file that can be used to customize the user interface.
-# custom_css = '/assets/test.css'
+# The CSS fle can be served from the public directory or via an external link.
```
Comment (willydouhard) [inline backend/chainlit/config.py:68]:
```suggestion
# The CSS file can be served from the public directory or via an external link.
```

Typo, almost there!

Comment (onepointconsulting) [reply backend/chainlit/config.py]: Should be fixed. Please check.

**Review by willydouhard — APPROVED** [review]

**Pull request merged**